src/main/plugin/iso19139.ca.HNAP/schematron/schematron-rules-common.sch

```diff
@@ -57,6 +57,33 @@
     </xsl:for-each>
   </xsl:function>
 
+  <xsl:key name="groupKey"
+    match="*//gmd:transferOptions/gmd:MD_DigitalTransferOptions/gmd:onLine/gmd:CI_OnlineResource[gmd:linkage/gmd:URL]"
+    use="concat(gmd:linkage/gmd:URL/text(), '|', gmd:description/gco:CharacterString/text())"/>
+
+  <xsl:function xmlns:sch="http://purl.oclc.org/dsdl/schematron"
+    name="geonet:getDuplicateResources"
+    as="xs:string">
+    <xsl:param name="elements" as="element()*"/>
+
+    <xsl:variable name="duplicateResources">
+      <xsl:for-each-group select="$elements"
+        group-by="concat(gmd:linkage/gmd:URL/text(), '|', gmd:description/gco:CharacterString/text())">
+
+        <xsl:if test="count(current-group()) &gt; 1">
+          <xsl:if test="exists(tokenize(current-group()[1]/gmd:description/gco:CharacterString/text(), ';')[3])">
+            <xsl:value-of select="concat(' ',tokenize(current-group()[1]/gmd:description/gco:CharacterString/text(), ',')[3], ';')"/>
```

Comment:
You have
`text(), ',')[3]`
But I believe the items are tokenized by a` ";"`?  Are you sure this works?

Reply:
It's a small bug. It doesn't work after I refresh my browser and restarted GN server. Let's just use semi colon instead of any comma here.